rest/model/build.go

```diff
@@ -22,36 +22,66 @@ var (
 
 // APIBuild is the model to be returned by the API whenever builds are fetched.
 type APIBuild struct {
-	Id                  *string        `json:"_id"`
-	ProjectId           *string        `json:"project_id"`
-	ProjectIdentifier   *string        `json:"project_identifier"`
-	CreateTime          *time.Time     `json:"create_time"`
-	StartTime           *time.Time     `json:"start_time"`
-	FinishTime          *time.Time     `json:"finish_time"`
-	Version             *string        `json:"version"`
-	Revision            *string        `json:"git_hash"`
-	BuildVariant        *string        `json:"build_variant"`
-	Status              *string        `json:"status"`
-	Activated           bool           `json:"activated"`
-	ActivatedBy         *string        `json:"activated_by"`
-	ActivatedTime       *time.Time     `json:"activated_time"`
-	RevisionOrderNumber int            `json:"order"`
-	TaskCache           []APITaskCache `json:"task_cache,omitempty"`
+	Id *string `json:"_id"`
+	// The identifier of the project this build represents
+	ProjectId         *string `json:"project_id"`
+	ProjectIdentifier *string `json:"project_identifier"`
+	// Time at which build was created
+	CreateTime *time.Time `json:"create_time"`
+	// Time at which build started running tasks
+	StartTime *time.Time `json:"start_time"`
+	// Time at which build finished running all tasks
+	FinishTime *time.Time `json:"finish_time"`
+	// The version this build is running tasks for
+	Version *string `json:"version"`
+	// Hash of the revision on which this build is running
+	Revision *string `json:"git_hash"`
+	// Build distro and architecture information
```

Comment:
I don't really understand this doc. What does it mean? I thought this would be the same as the `DisplayName`?